README.md

```diff
@@ -63,10 +63,38 @@ The `proxy` parameter is optional, and it indicates what HTTP proxy should be
 used to connect to the AWX API. If this parameter is not specified, or if it is
 empty, then the connection will be direct to the AWX server, without a proxy.
 
-The `credentialsRef` parameter is a reference to the [Kubernetes
+The `credentials` section contains the user name and password used to connect to
+the AWX API. For example:
+
+```yaml
+credentials:
+  username: autoheal
+  password: ...
+```
+
+Note that if you store the credentials in the configuration file then it needs
+to be protected very carefully. For example, you can create a separate file for
+the the credentials, give it restricted permissions, and then load it using the
```

Comment:
double word `the`.
I prefer to discourage users from using this in production as secrets are better suited.

Reply:
In the next version of the pull request I am explicitly saying what is the recommended mechanism for production environments.